bigquery_etl/glam/templates/histogram_bucket_counts_v1.sql

```diff
@@ -7,7 +7,9 @@ WITH
         source_table,
         "all_combos",
         cubed_attributes,
-        attribute_combinations
+        attribute_combinations,
+        true,
+        "table.os = 'Windows' AND channel = 'release' AS sampled"
```

Comment:
The initial reason was because the code usually only processes the last 3 versions of data and the current versions - 3 are all over 119. That said, I'll keep it in case the code gets used for back-filling.